2021-12-03 01:16:25,896 INFO    MainThread:50288 [wandb_setup.py:_flush():71] setting env: {}
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_init.py:_log_setup():371] Logging user logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_011625-28pvp9qs/logs/debug.log
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_011625-28pvp9qs/logs/debug-internal.log
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_init.py:init():404] calling init triggers
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'max_epochs': 150, 'batch_size': 100}
2021-12-03 01:16:25,897 INFO    MainThread:50288 [wandb_init.py:init():449] starting backend
2021-12-03 01:16:25,897 INFO    MainThread:50288 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-03 01:16:25,911 INFO    MainThread:50288 [backend.py:ensure_launched():199] starting backend process...
2021-12-03 01:16:25,918 INFO    MainThread:50288 [backend.py:ensure_launched():205] started backend process with pid: 50308
2021-12-03 01:16:25,925 INFO    MainThread:50288 [wandb_init.py:init():458] backend started and connected
2021-12-03 01:16:25,928 INFO    MainThread:50288 [wandb_init.py:init():519] updated telemetry
2021-12-03 01:16:25,943 INFO    MainThread:50288 [wandb_init.py:init():549] communicating current version
2021-12-03 01:16:27,018 INFO    MainThread:50288 [wandb_init.py:init():554] got version response 
2021-12-03 01:16:27,018 INFO    MainThread:50288 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-03 01:16:27,248 INFO    MainThread:50288 [wandb_init.py:init():592] starting run threads in backend
2021-12-03 01:16:30,648 INFO    MainThread:50288 [wandb_run.py:_console_start():1816] atexit reg
2021-12-03 01:16:30,649 INFO    MainThread:50288 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-03 01:16:30,649 INFO    MainThread:50288 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-03 01:16:30,651 INFO    MainThread:50288 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-03 01:16:30,651 INFO    MainThread:50288 [wandb_init.py:init():619] run started, returning control to user process
2021-12-03 01:16:30,651 INFO    MainThread:50288 [wandb_watch.py:watch():43] Watching
2021-12-03 01:16:30,657 INFO    MainThread:50288 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-03 01:16:30,657 INFO    MainThread:50288 [wandb_run.py:_restore():1758] restore
2021-12-03 01:16:33,185 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 01:16:33,416 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 01:16:33,906 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7762
}

2021-12-03 01:16:34,008 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7788
}

2021-12-03 01:16:34,116 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7788
}

2021-12-03 01:16:34,222 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:34,327 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:34,433 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:34,539 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:34,643 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:35,069 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}

2021-12-03 01:16:35,564 INFO    MainThread:50288 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7788
  total_bytes: 7788
}
local_info {
}

2021-12-03 01:16:36,886 INFO    MainThread:50288 [wandb_run.py:_append_files():2184] logging synced files
